### PR TITLE
add timestamp function

### PR DESCRIPTION
**Pull request by SebastienValla**

### CODE REVIEW

**Review by Thykof — APPROVED** [review]
LGTM

**Review by 0xMazout — APPROVED** [review]
Tested & Approved 👍

File: astester.imports.js [hunk 1]
```diff
@@ -172,6 +172,9 @@ export function local(memory) {
         ledger.set(newAddress._value, newAddressLedger);
         return newString(newAddress._value);
       },
+      assembly_script_get_time(){
+        return BigInt(Date.now());
```
Comment (gregLibert) [inline astester.imports.js:176]: You should force the conversion to an u64 value yourself.

```suggestion
        return BigInt.asUintN(64, Date.now());  
```

**Review by gregLibert — CHANGES_REQUESTED** [review]
:mag: you should handle the proper length of your returned arguments type yourself.

**Review by gregLibert — APPROVED** [review]
:+1:

**Pull request merged**